style.css

```diff
@@ -806,11 +788,321 @@ header {
 }
 
 .fmargin {
-  position: absolute;
+  order: 5;
   width: 134px;
   height: 5px;
   left: calc(50% - 134px / 2 - 0.5px);
   bottom: 8px;
   border-radius: 10px;
   background-image: url("./Images/Shape.png");
 }
+
+/* media query section */
+
+@media (min-width: 768px) {
+  header {
+    width: 100%;
+    height: 72px;
+  }
+
+  .bar2 {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .bar2 .logo {
+    width: 116px;
+    height: 44px;
+    left: 142px;
+    top: 16px;
+    gap: 10px;
+  }
+
+  .bar2 .normalb {
+    display: none;
+  }
+
+  .bar2 .burger {
+    margin-right: 6rem;
+    display: flex;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    background-color: #3c3a39;
+    border: none;
+    align-items: center;
+    top: 16px;
+  }
+
+  .bar2 .burger a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 15%;
+    height: 40px;
+    flex: none;
+    flex-grow: 0;
+  }
+
+  .bar2 .burger a p {
+    text-decoration: none;
+    color: #fff;
+    width: 63px;
+    height: 20px;
+    font-family: Inter, serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+  }
+
+  #name {
+    font-size: 20px;
+  }
+
+  /* main section */
+  .continer {
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    height: 870px;
+    background-image: url("./Images/Illustration-header.png");
+    background-position: top;
+    background-position-x: right;
+    background-size: 45% 90%;
+    padding-left: 0;
+  }
+
+  .textcontainer {
+    order: 1;
+    width: 40%;
+    height: unset;
+  }
+
+  .title {
+    width: 100%;
+  }
+
+  .paragraph {
+    width: 100%;
+  }
+
+  .continer .social {
+    flex-direction: column;
+    height: 12rem;
+    order: 0;
+  }
+
+  /* Section works header */
+  .works {
+    align-items: center;
+    padding: 120px 24px;
+    gap: 114px;
+    width: 100%;
+  }
+
+  #wheader {
+    flex-direction: row;
+    margin-top: 135px;
+    width: 80%;
+    height: 52px;
+  }
+
+  #line {
+    width: 70%;
+    order: 1;
+  }
+
+  #wintro {
+    flex-direction: row;
+    padding: 0;
+    gap: 24px;
+  }
+
```

Comment:
- [ ] Kindly make sure that the width and margins of the element on the page match that of the design template as much as possible.
Your design
![Capture1](https://user-images.githubusercontent.com/85095702/171739326-1e9170fc-0d63-4438-b2a4-a8e24f206f9a.jpg)
![Capture2](https://user-images.githubusercontent.com/85095702/171739487-d4e4a7cf-1bd3-4eb4-a093-d8f50289c415.jpg)

Figma design
![figma](https://user-images.githubusercontent.com/85095702/171739702-215654c6-7fcf-4824-a03f-a7097bf8a455.jpg)